posts/2023-09-19-23.0.0.9.adoc

```diff
@@ -18,9 +18,9 @@ Laura Cowen <https://github.com/lauracowen>
 :url-about: /
 //Blank line here is necessary before starting the body of the post.
 
-Open Liberty's support for Spring Boot 3.0 means that you can use Spring Boot 3.x capabilities in your Liberty applications. You can also now authenticate your OpenID Connect clients using the Private Key method instead of client secrets, and if you use LTPA/JWT cookies, you can now use different cookies for different applications according to their context root.
+You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the Private Key method instead of client secrets, and if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.
```

Comment:
```suggestion
You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the more secure Private Key method instead of client secrets, and, if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.
```